lib/twitter/rest/request.rb

```diff
@@ -26,6 +27,7 @@ def initialize(client, request_method, path, options = {})
         @client = client
         @uri = Addressable::URI.parse(path.start_with?('http') ? path : BASE_URL + path)
         set_multipart_options!(request_method, options)
+        timeout_options!(options)
```

Comment:
another trivial thing- this could be `set_timeout_options!` to be consistent with the method above.